scripts/run-integration-template.bash

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+#
+
+pushd () {
+    command pushd "$@" > /dev/null
+}
+
+popd () {
+    command popd "$@" > /dev/null
+}
+
+INTEGRATION=$1
+shift
+
+NANGO_SECRET_KEY_DEV=$(grep NANGO_SECRET_KEY_DEV .env | cut -d '=' -f2)
+
+rm -rf temp
```

Comment:
nitpick: I would use a less generic name for the temp directory, something like `tmp-run-integration-template`. Or even putting it in the OS `/tmp` dir. Up to you though.

Reply:
Updated